web/src/compose_recipient.ts

```diff
@@ -275,6 +275,10 @@ function on_hidden_callback(): void {
         // don't do anything.
         return;
     }
+    if (compose_state.is_item_clicked()) {
```

Comment:
Can you explain why we need to rely on `is_item_clicked` here? Wouldn't `s_processing_forward_message` remain always `true` if the dropdown was closed without a click?